### ISSUE TITLE
Approved programs bug for DWP/MFIP results same day approvals

### ISSUE THREAD

**Issue by IlseFerris** [MN-Script-Team/DHS-MAXIS-Scripts#2953]
User reporting error when DWP is closed for CM plus one, and MFIP is approved for CM plus one.  Script attempts to read approved CASH results, but the approved CASH results for DWP are autoclosed. Throws "line 1 column 1: type mismatch 'cdate'. This is most common for DWP, but could happen with other CASH programs as well. Suggesting a bypass for cash programs that autoclose, or cases that autoclose in general.

Possible related issue #1771

**Comment by C-Love** [comment]
Yep - I think we can just write some error proofing to look for that message pop up when navigating to elig.

**Comment by C-Love** [comment]
This should be considered when addressing issue #3012 (merging Approved/Denied/Closed programs scripts).

**Comment by C-Love** [comment]
Group discussion to fix now.
